x-pack/plugins/maps/public/shared/components/metric_select.js

```diff
@@ -6,15 +6,30 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { i18n } from '@kbn/i18n';
 import { EuiComboBox } from '@elastic/eui';
 
 const AGG_OPTIONS = [
-  { label: 'Average', value: 'avg' },
-  { label: 'Count', value: 'count' },
-  { label: 'Max', value: 'max' },
-  { label: 'Min', value: 'min' },
-  { label: 'Sum', value: 'sum' },
+  { label: i18n.translate('xpack.maps.metricSelect.averageDropDownOptionLabel', {
+    defaultMessage: 'Average'
+  }),
+  value: 'avg' },
+  { label: i18n.translate('xpack.maps.metricSelect.count.averageDropDownOptionLabel', {
```

Comment:
`count.averageDropDownOptionLabel` does not make sense. 

How about `countDropDownOptionLabel`?

Same for max, min, and sum